### PR TITLE
[12.0][IMP] openupgrade_records: show noupdate xmlid changes between versions

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
This way, although the xmlid is the same between versions, if it has a noupdate change, from True to False, or from False to True, it will show the two old and new xmlids in order to detect this cases.

Updating a module doesn't change the noupdate flag from `ir.model.data`. It has to be changed manually in the migration scripts.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I see no direct message is put that the noupdate has been switched, so it will be difficult to recognize it, isn't it?

**Comment by MiquelRForgeFlow** [comment]
> so it will be difficult to recognize it, isn't it?

I don't think so. But let's wait for the update analysis and then say me again that 😏

**Comment by MiquelRForgeFlow** [comment]
> direct message is put that the noupdate has been switched

Now it's put.

File: odoo/addons/openupgrade_records/lib/compare.py [hunk 1]
```diff
@@ -304,10 +304,16 @@ def match(match_fields, match_type='direct'):
                 else:
                     column['domain'] = False
                     found['domain'] = False
+                column['noupdate_switched'] = False
+                if column['noupdate'] != found['noupdate']:
+                    found['noupdate_switched'] = True
+                else:
+                    found['noupdate_switched'] = False
```
Comment (pedrobaeza) [inline odoo/addons/openupgrade_records/lib/compare.py]:
```suggestion
                    found['noupdate_switched'] = column['noupdate'] != found['noupdate']
```

Comment (MiquelRForgeFlow) [reply odoo/addons/openupgrade_records/lib/compare.py]: right 😁

**Review by pedrobaeza — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**